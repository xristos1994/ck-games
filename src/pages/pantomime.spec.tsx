import React from 'react';
import renderer from 'react-test-renderer';
import PantomimePage from './pantomime';
import EL_PantomimePage from './el/pantomime';
import EN_PantomimePage from './en/pantomime';

const Pantomime = () => {
  return (
    <div>
      Pantomime
    </div>
  );
};

jest.mock('@page-components/', () => ({
  __esModule: true,
  Pantomime: Pantomime
}));

describe('pages/Home', () => {
  it('renders correctly', () => {
    let tree = renderer.create(
      <PantomimePage />
    ).toJSON();
    expect(tree).toMatchSnapshot();

    tree = renderer.create(
      <EL_PantomimePage />
    ).toJSON();
    expect(tree).toMatchSnapshot();

    tree = renderer.create(
      <EN_PantomimePage />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
