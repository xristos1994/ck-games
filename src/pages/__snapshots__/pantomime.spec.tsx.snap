// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages/Home renders correctly 1`] = `
<div>
  Pantomime
</div>
`;

exports[`pages/Home renders correctly 2`] = `
<div>
  Pantomime
</div>
`;

exports[`pages/Home renders correctly 3`] = `
<div>
  Pantomime
</div>
`;
